Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN \
     sleep 1 && \
     ./mvnw package -Pprod -DskipTests && \
     mv /code/target/*.war /jhipster-registry.war && \
+    mkdir /target && \
```

Comment:
As we use multi stage, these 2 lines must be in the 2nd part -> FROM openjdk:8-jre-alpine, instead of the 1st one

Reply:
@pascalgrimaud I'm not sure about this. It seems the problem is with building the jar. Which is lacking the permissions to automatically create the directory.
I never had the issue myself but it seems @hufon has tested it.

Reply:
I agree, because now it's the same :
```
docker run jhipster/jhipster-registry:master /bin/sh -c 'ls -la /target'
ls: /target: No such file or directory
```